qiita_db/processing_job.py

```diff
@@ -186,13 +253,13 @@ def status(self):
             qdb.sql_connection.TRN.add(sql, [self.id])
             return qdb.sql_connection.TRN.execute_fetchlast()
 
-    @status.setter
-    def status(self, value):
+    def _set_status(self, value):
```

Comment:
Note that I changed the this from being a property setter to be an internal function - so the tests for the property setter still applies (I renamed them)